### ISSUE TITLE
[0.3.7] .ping returns streaming JSON in the browser

### ISSUE THREAD

**Issue by Dignifiedquire** [ipfs/js-ipfs-api#86]
Request:

```
http://127.0.0.1:56191/api/v0/ping?n=1&arg=QmWkMgYviPiHjfCdKkG4mmYotr5zJ
```

Response:

```
{
  "Success": false,
  "Time": 0,
  "Text": "PING QmWkMgYviPiHjfCdKkG4mmYotr5zJn3vVpMpYFCaezFQVH."
}{
  "Success": true,
  "Time": 1341453,
  "Text": ""
}{
  "Success": false,
  "Time": 0,
  "Text": "Average latency: 1.34ms"
}
```

**Comment by diasdavid** [comment]
Does it continue to stream? Or was only 3 pings?

@whyrusleeping thoughts?

**Comment by Dignifiedquire** [comment]
That's the whole response for that request (it ends then)

**Comment by whyrusleeping** [comment]
are the quotes actually included in the response?

On latest master there should be a newline between objects, which makes this easier to parse.
The format is ndjson, we dont want to return an 'array' because parsers will wait until the array is closed before finishing, and we want to act on each object as it arrives.

**Comment by Dignifiedquire** [comment]
> are the quotes actually included in the response?

No sorry the surrounding quotes are not in the response

**Comment by diasdavid** [comment]
solved with 0.3.9
